hedgehog/src/Hedgehog/Internal/Runner.hs

```diff
@@ -394,6 +401,35 @@ checkParallel =
           Nothing
       }
 
+-- | Run list of tests.
+--
+-- Given a number of separate modules all defining test funtion named `tests`,
+-- the individual module level tests can be collected together and run in
+--  sequence (one module of tests at a time) and the funcion will will exit
```

Comment:
Nitpick: there seems to be an extra space between `sequence`.
Nitpick: `will` occurs two times in a row on that sentence.

Reply:
Yeah, this PR will not go ahead as it is anyway.